### DISCUSSION TITLE
原因のAI入力の無効化

### DISCUSSION THREAD

**Discussion by gr1m0h** [topotal/waroom-roadmap#70] [Feature Request]
原因を手入力した後にAI入力によって置き換わってしまったので再度手入力で修正するということが発生した。
そのため、ステートドキュメント・ポストモーテム同様、AIによる入力を無効化できるようにしたい。

![image (1)](https://github.com/topotal/waroom-roadmap/assets/16541988/25433425-e2f2-43d9-8f7e-25c6894258a0)

**Comment by sawa-zen** [comment]
@gr1m0h 
ありがとうございます！僕らの想定では「手動入力されていた場合はAIによる上書きはされない」はずでしたが、現在確認したところ上書きされてしまっていたのでこちら修正しようと思います。ご報告大変たすかりました 🙏🏻 

ちなみに「手動入力されていた場合はAIによる上書きはされない」の仕様に正しく修正された先で AI による原因ラベルの入力機能の無効化設定は必要そうでしょうか？

**Reply by gr1m0h** [reply]
@sawa-zen 

> 僕らの想定では「手動入力されていた場合はAIによる上書きはされない」はずでしたが、現在確認したところ上書きされてしまっていたのでこちら修正しようと思います。ご報告大変たすかりました 🙏🏻

承知です！

> ちなみに「手動入力されていた場合はAIによる上書きはされない」の仕様に正しく修正された先で AI による原因ラベルの入力機能の無効化設定は必要そうでしょうか？

不要に思います！

**Reply by sawa-zen** [reply]
@gr1m0h 
承知いたしました！修正のリリースが完了次第またご連絡いたします 🙏🏻

**Reply by sawa-zen** [reply]
@gr1m0h 
こちら修正のリリースが完了しました！手動入力されていた場合はAIによる上書きはされなくなりました 🙏🏻

**Comment by sawa-zen** [comment]
こちら修正が完了しましたのでクローズします。何かまた問題があればお知らせください。
